tests/run/cdef_class_dataclass.pyx

```diff
@@ -224,6 +224,25 @@ cdef class TestFrozen:
     """
     a: double = 2.0
 
+@dataclasses(kw_only=True)
```

Comment:
```suggestion
@dataclass(kw_only=True)
```

You can run the tests in this file locally on your PC with `python runtest.py cdef_class_dataclass` just to confirm that it works.